HIP/hip-1126.md

```diff
@@ -0,0 +1,52 @@
+---
+hip: 1126
+title: "Delete Allowance without Token Association"
+author: Michael Heinrichs <@netopyr>
+working-group:
+requested-by: GitHub issue https://github.com/hiero-ledger/hiero-consensus-node/issues/17441
+type: "Standards Track"
+category: "Service"
+needs-council-approval: "Yes"
+status: "Draft"
+created: 2025-02-17
+discussions-to: https://github.com/hiero-ledger/hiero-consensus-node/issues/17441
+updated: 2025-02-17
+requires: 
+replaces: 
+superseded-by: 
+---
+
+## Abstract
+To remove an allowance for a token, the token must be associated with the account.
+This is an unnecessary burden with no advantages.
+This HIP proposes to allow the removal of allowances without requiring the token to be associated.
+
+## Motivation
+If a user wants to remove an allowance for a token without an association, one has to associate the token, remove the allowance, and dissociate the token again.
+This is a cumbersome and expensive process with no benefits.
+A user may decide to keep the allowances instead, which wastes resources and risks hitting the limit of allowed allowances.
+
+While it makes sense to associate the token before giving a non-zero allowance, it does not make sense that the token must be associated in order for the allowance to be removed.
+
+## Rationale
+The execution of `CryptoDeleteAllowance` does not require the owner to have an association with the token.
```

Comment:
Perhaps we can elaborate that in order to remove allowance we use `CryptoDeleteAllowance` for NFTs only and `CryptoApproveAllowance` with an amount of `0` for fungible tokens, because initially I thought we should use `CryptoDeleteAllowance` for both.

Reply:
Added clarification